reporting_app/templates/proc_report.html

```diff
@@ -1,91 +1,75 @@
+        </div>
+        <div>
             Status:
             {% if proc.status %}
             {{ proc.status }}
             {% else %}
             inactive
             {% endif %}
-        </li>
-    </ul>
+        </div>
+    </div>
     <br/>
 
-    <div id="{{ proc.proc_id }}"></div>
+    <div id="timeline_{{ proc.proc_id }}"></div>
 
     <script type="text/javascript">
-        var container = document.getElementById('{{ proc.proc_id }}');
-        var stages = {{ proc.stages|list|safe }};
-        var stage_names = stages.map(function(p){
-            if (p.date_finished){
-                return {
-                    content: p.stage_name,
-                    'start': moment(p.date_started, "DD-MM-YYYY HH:mm:SS").format('YYYY-MM-DD HH:mm:SS'),
-                    'end': moment(p.date_finished, "DD-MM-YYYY HH:mm:SS").format('YYYY-MM-DD HH:mm:SS')
-                }
-            } else {
-                return {
-                    'start': moment(p.date_started, "DD-MM-YYYY HH:mm:SS").format('YYYY-MM-DD HH:mm:SS'),
-                    'style': "color: red; background-color: pink;"
-                }
-            }
-        });
-
-
-        var items = new vis.DataSet(stage_names);
-        startDate = (moment('{{ proc._created }}', "DD-MM-YYYY HH:mm:SS").format('YYYY-MM-DD HH:mm:SS'))
-        startMargin = moment(startDate).subtract(1, 'days')
-        endMargin = moment(startDate).add(6, 'days')
-        var options = {
-            min: startMargin,
-            max: endMargin
-        }
-        var timeline = new vis.Timeline(container, items, options);
+        stage_timeline("timeline_{{ proc.proc_id }}", {{ proc|safe }});
     </script>
     <br/>
 {%- endmacro %}
 
 
 {% macro proc_report(procs) -%}
-{# args:
-    procs (list[dict]) - analysis_driver_procs from the Rest API.
-#}
-    <script src="//cdnjs.cloudflare.com/ajax/libs/vis/4.16.1/vis.min.js"></script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/moment.js/2.13.0/moment.min.js"></script>
+{# procs (list[dict]) - analysis_driver_procs from the Rest API. #}
 
-    <h2>
-        <a data-toggle="collapse" href="#analysis_driver_proc_tabs">
-            Analysis Driver processes
-        </a>
-    </h2>
-    <div id="analysis_driver_proc_tabs" class="tab-content collapse">
-        <p>Number of times processed: {{ procs|length }}</p>
-        {% if procs %}
-            <ul class="nav nav-tabs" id="analysis_driver_proc_ul">
-                <li class="nav active"><a data-target="#analysis_driver_proc_1_tab" data-toggle="tab">{{ procs[0]._created }}</a></li>
-                {% for proc in procs[1:] %}
-                    <li class="nav"><a data-target="#analysis_driver_proc_{{ loop.index + 1 }}_tab" data-toggle="tab">{{ proc._created }}</a></li>
-                {% endfor %}
-            </ul>
-
-            <div id="analysis_driver_proc_1_tab" class="tab-pane fade in active"> {{ proc_timeline(procs[0]) }} </div>
+<h2>Analysis Driver processes</h2>
```

Comment:
That was not intended - I wanted to keep it collapsible, but show it by default when the page loads. Turns out removing the 'collapse' from the `analysis_driver_proc_tabs` div does it.